packages/input-edit/src/main.js

```diff
@@ -10,10 +10,15 @@ import InputEdit from './components/InputEdit/InputEditContainer'
 
 const packageName = 'input-edit'
 
-const initApp = (id, input, events, publicPath) => {
+const initApp = (id, input, events = {}, publicPath) => {
   const content = <InputEdit/>
 
   const store = appFactory.createStore(reducers, sagas, input, packageName)
+  actionEmitter.addToStore(store)
+  actions.addToStore(store)
+  notifier.addToStore(store, false)
+  errorLogging.addToStore(store, false)
+  externalEvents.addToStore(store, events)
```

Comment:
externalEvents is not used yet? I think there should be more code that it works properly.
see:
https://github.com/tocco/tocco-client/blob/master/packages/entity-detail/src/main.js#L107

Reply:
When initially developing I got an error from the entity-browser when I didn't include this line and then opened my action. I couldn't reproduce it now though, so I'm just removing it again.